Formula/ksh.rb

```diff
@@ -1,42 +1,29 @@
 class Ksh < Formula
-  desc "KornShell, ksh93"
+  desc "KornShell, ksh93u+m"
   homepage "https://github.com/ksh93/ksh#readme"
-  url "https://github.com/ksh93/ksh/archive/refs/tags/v1.0.0-beta.2.tar.gz"
-  sha256 "d8678d23c3c9633a03e4fc895e604cdd0af2ff006d0268579b3a29beddfb8463"
-  license "EPL-1.0"
-
-  bottle do
-    sha256 cellar: :any_skip_relocation, arm64_monterey: "8fa4ca19495438d415fae8d81de8efdb45148f4d7e5733a5a294975f9e889b18"
-    sha256 cellar: :any_skip_relocation, arm64_big_sur:  "ade0f9307c006c9833a85a4c046019668e327197ebe0220c8ed74fa59958f090"
-    sha256 cellar: :any_skip_relocation, monterey:       "100c44e8c7bfad8c63fc3eb8ee2afc9a5c10f33481c71f4283d6720ce78e0ba4"
-    sha256 cellar: :any_skip_relocation, big_sur:        "92ca5f26f79c9ca9637331d83b1e1521ab3e1c772d9e2d363c8353fb2c568a61"
-    sha256 cellar: :any_skip_relocation, catalina:       "ea7be886a6acee55713ca673ce1578a1303389fb2a964734c38137d3610d7f2b"
-    sha256 cellar: :any_skip_relocation, mojave:         "3bc3469d43fba904b3045722d43bb52444f88c2e6745af977bae9b52d1f0090e"
-    sha256 cellar: :any_skip_relocation, high_sierra:    "733e1c6bdd05054bf8d0097a6ae9ea2ca21e74b4676df7b424d4b9f43078afd4"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "8a6f8d5a96eaf9d91da32c58e453d7aacc4f47acea57f6a2b3db7cc108bbcd1f"
-  end
+  url "https://github.com/ksh93/ksh/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "54f1fcfea77ec55a639e08f0fe66a1fc918c762f176cad917ee93b44f511e1ef"
+  license "EPL-2.0"
+  head "https://github.com/ksh93/ksh.git", branch: "dev"
 
   def install
-    system "bin/package", "make"
-    system "sh", "-c", "mv arch/\"$(bin/package)\" output"
-    bin.install "output/bin/ksh"
+    system "bin/package", "verbose", "make"
+    system "bin/package", "verbose", "install", prefix
     bin.install_symlink "ksh" => "ksh93"
     bin.install_symlink "ksh" => "rksh"
-    bin.install "output/bin/shcomp"
-    man.install "output/man/man1/sh.1" => "ksh.1"
-    man.install_symlink "ksh.1" => "ksh93.1"
-    man.install_symlink "ksh.1" => "rksh.1"
-    doc.install "src/cmd/ksh93/README"
+    man1.install_symlink "ksh.1" => "ksh93.1"
+    man1.install_symlink "ksh.1" => "rksh.1"
     doc.install "ANNOUNCE"
-    doc.install "NEWS"
     doc.install "COPYRIGHT"
     doc.install "LICENSE.md" => "LICENSE"
+    doc.install "NEWS"
```

Comment:
This should all be automatic

Reply:
Of those at the top-level ANNOUNCE wasn't automatic, I removed the others (NEWS, COPYRIGHT, LICENSE.md).